docs/getting-started/user-guides/dcrwallet-setup.md

```diff
@@ -0,0 +1,116 @@
+# **dcrwallet Setup Guide**
+
+Last updated for v0.8.2.
+
+This guide is intended to help you setup the `dcrwallet` application using [startup flags](/getting-started/startup-basics.md#startup-command-flags). 
+
+**Prerequisites:**
+
+- Use either the latest [dcrinstall](/getting-started/install-guide.md#dcrinstall) or [Binary Release archive](/getting-started/install-guide.md#binary-releases) archive to install `dcrwallet`.
+- Review how the launch commands for the Command Prompt (Windows) and Bash (OSX/Linux) shells differ [here](/getting-started/operating-systems.md#command-line-differences-across-operating-systems).
+- [Setup dcrd](/getting-started/user-guides/dcrd-setup.md) and have it running in the background.
+
+---
+
+`dcrwallet` is the daemon that handles Decred wallet functionality for a single user. It manages all of your accounts, addresses, and transactions; tracks balances across addresses; and allows stakeholders to participate in Proof-of-Stake voting.
+
+---
+
+## **Critical Information**
+
+During the creation process for your wallet, you will be given a sequence of 33 words known as a seed phrase. This seed phrase is essentially the private key for your wallet. You will be able to use this seed phrase to restore your private keys, transaction history, and balances using any Decred wallet on any computer. 
+
+This ultimately means that *anyone* can use your seed phrase to restore your private keys, transaction history, and balances to a Decred wallet on their computer. For this reason, it is of utmost importance to keep your seed phrase safe. Treat this seed the same way you would treat a physical key to a safe. If you lose your seed phrase, you permanently lose access to your wallet and all funds within it. It cannot be recovered by anyone, including the Decred developers. It is recommended you write it down on paper and store that somewhere secure. If you decide to keep it on your computer, it would be best to keep it in an encrypted document (do not forget the password) in case the file or your computer is stolen.
+
+Every seed phrase is also associated with a 64 character seed hex. A seed hex can be decoded and converted to a seed phrase for importing into a new wallet using [this tool](https://github.com/davecgh/dcrseedhextowords), written by one of the Decred developers. It is also important to keep your seed hex secure.
```

Comment:
dcrwallet can directly import the hex seed.  I added that feature pretty early on so no need for davec's too when using dcrwallet.